tsconfig.base.json

```
@@ -17,7 +17,8 @@
     "baseUrl": ".",
     "paths": {
       "twenty-emails": ["packages/twenty-emails/src/index.ts"],
-      "twenty-ui": ["packages/twenty-ui/src/index.ts"]
+      "twenty-ui": ["packages/twenty-ui/src/index.ts"],
+      "twenty-shared": ["packages/twenty-shared/dist"]
```

```suggestion
      "twenty-shared": ["packages/twenty-shared/src/index.ts"]
```
Please target the barrel file here `packages/twenty-shared/src/index.ts` instead of targeting dist